common/static/common/js/spec/discussion/view/discussion_thread_view_spec.js

```diff
@@ -276,127 +271,13 @@
-                });
-                it('strips script tags appropriately', function() {
-                    var longMaliciousBody, maliciousAbbreviation;
-                    DiscussionViewSpecHelper.setNextResponseContent({
-                        resp_total: 0,
-                        children: []
-                    });
-                    longMaliciousBody = new Array(100).join(
-                        "<script>alert('Until they think warm days will never cease');</script>\n"
-                    );
-                    this.thread.set('body', longMaliciousBody);
-                    maliciousAbbreviation = DiscussionUtil.abbreviateString(this.thread.get('body'), 140);
-                    this.view.render();
-                    expect($('.post-body').html()).not.toEqual(maliciousAbbreviation);
-                    expect($('.post-body').text()).toEqual(maliciousAbbreviation);
-                    expect($('.post-body').html()).not.toContain('<script');
-                    this.view.expand();
-                    expect($('.post-body').html()).not.toEqual(longMaliciousBody);
-                    expect($('.post-body').text()).toEqual(longMaliciousBody);
-                    expect($('.post-body').html()).not.toContain('<script');
-                    this.view.collapse();
-                    expect($('.post-body').html()).not.toEqual(maliciousAbbreviation);
-                    expect($('.post-body').text()).toEqual(maliciousAbbreviation);
-                    return expect($('.post-body').html()).not.toContain('<script');
-                });
-                it('re-renders the show view correctly when leaving the edit view', function() {
-                    DiscussionViewSpecHelper.setNextResponseContent({
-                        resp_total: 0,
-                        children: []
-                    });
-                    this.view.render();
-                    this.view.expand();
-                    assertExpandedContentVisible(this.view, true);
-                    this.view.edit();
-                    assertContentVisible(this.view, '.edit-post-body', true);
-                    expect(this.view.$el.find('.post-actions-list').length).toBe(0);
-                    this.view.closeEditView(DiscussionSpecHelper.makeEventSpy());
-                    expect(this.view.$el.find('.edit-post-body').length).toBe(0);
-                    return assertContentVisible(this.view, '.post-actions-list', true);
-                });
-            });
-        });
         describe('for question threads', function() {
             var generateContent, renderTestCase;
             beforeEach(function() {
                 this.thread.set('thread_type', 'question');
                 this.view = new DiscussionThreadView({
                     model: this.thread,
                     el: $('#fixture-element'),
-                    mode: 'tab',
```

Comment:
Note that the default is "inline" if the mode isn't provided. I don't think that matters, but just FYI.